modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js

```diff
@@ -43,6 +48,42 @@ export default function ChangeTrackingWorkflowView({workflowData}) {
 
 						<ClayTable.Cell className="table-cell-expand">
 							{workflowData.assignedTo}
+
+							<ClayButton
+								className="ml-2"
+								displayType="secondary"
+								onClick={() =>
+									Liferay.Util.openModal({
+										center: true,
+										customEvents: [
+											{
+												name: `${namespace}workflowTaskUpdated`,
+												onEvent() {
+													const iframe = document.querySelector(
+														'.liferay-modal iframe'
+													);
+
+													iframe.contentWindow.location.reload();
+
+													setShowWorkflowSuccessMessage(
+														true
+													);
+												},
+											},
+										],
+										height:
+										workflowData.assignButton.modalHeight,
+										onOpen: () =>
+											setShowWorkflowSuccessMessage(false),
+										size: 'lg',
+										title: workflowData.assignButton.label,
+										url: workflowData.assignButton.href,
+									})
+								}
+								size="xs"
+							>
```

Comment:
Since this is the same code we use for the dropdown action, make this a function in the parent component and pass it in as props